docker/build_fake_dbs.sh

```diff
@@ -9,10 +9,10 @@ conda activate icubam
 pwd
 ls -a
 echo "Build test db"
-PYTHONPATH=. python scripts/populate_db_fake.py --config=resources/config.toml --mode=dev
+PYTHONPATH=. python scripts/populate_db_fake.py --config=resources/config_dev.toml
 
 echo "Build icubam db"
-PYTHONPATH=. python scripts/populate_db_fake.py --config=resources/config.toml --mode=prod
+PYTHONPATH=. python scripts/populate_db_fake.py --config=resources/config_prod.toml
```

Comment:
```suggestion
PYTHONPATH=. python scripts/populate_db_fake.py --config=resources/config.toml
```

Reply:
Thanks. indeed I noticed and changed already on my side, fixing the different shell scripts for starting docker currently